### ISSUE TITLE
Can't use KeeAgent with Tabby SSH Client

### ISSUE THREAD

**Issue by Stefomat** [dlech/KeeAgent#384]
I use KeePass 2.53.1 with KeeAgent 0.13.4. This works wonderfully for PuTTY and WinSCP. Unfortunately, the SSH client Tabby does not recognise KeeAgent. The normal Pageant in version 0.77 is recognised without problems, but KeeAgent is not. Does KeeAgent behave differently from PuTTY Pageant in some respects? How can I debug this problem? Is it a Tabby problem or a KeeAgent problem? Thanks!

**Comment by dlech** [comment]
You can run KeeAgent from source using Visual Studio to debug the problem.

**Comment by dlech** [comment]
Did you manually select the agent connection type? Selecting Pageant seems to work but Named Pipe had some issues until I fixed #362.

![image](https://user-images.githubusercontent.com/963645/227792948-cf670bca-4617-40d3-afb0-782c4031f28b.png)

**Comment by Stefomat** [comment]
I can reproduce the following behaviour:
Agent type set to Automatic:
No agent is found with KeeAgent started ("SSH ! Agent auth selected, but no running agent is detected").
Without KeeAgent started, password Auth is used.
Agent type set to Pageant:
With KeeAgent started, the agent is found and used.
Without KeeAgent started, it does not fall back to password Auth, Auth fails completely.
Agent type on Named Pipe:
Doesn't work at all, probably because I don't know which path to specify.

I have not yet been able to debug the problem with VSCode, due to lack of time and skills.

**Comment by dlech** [comment]
Did you try with v0.13.5 (just released yesterday)?

For the named pipe option, you have to enable **Windows OpenSSH** in the [global options](https://keeagent.readthedocs.io/en/latest/usage/options.html#global-options). You don't have to specify a path - the default one is correct.

**Comment by Stefomat** [comment]
Yes i have also tried with v0.13.5, thats the version my last comment is based on.

**Comment by wiesl** [comment]
For me with latest versions (KeeAgent v0.13.8 and Tabby v1.0.223):
I have OpenSSH agent enabled in KeeAgent.

Depending on selected Agent type in Tabby:
- Pageant: does NOT work
  - Error Message:
```
SSH   !  Failed to authenticate using agent: Error: IO(Custom { kind: UnexpectedEof, error: "early eof" })
 X  Authentication rejected
```
- Named pipe: OK with default pipe \\\\.\pipe\openssh-ssh-agent
- Auto: OK (guess it will use OpenSSH agent first)

Any ideas?

**Comment by wiesl** [comment]
Also latest Putty Agent V0.83 doesn't work with "Pageant Mode". So looks like a problem in Tabby.

OpenSSH mode is also ok:
pageant.exe --openssh-config pageant.conf

pageant.conf looks then like:
//./pipe/pageant.username.verylongcustomhexid
=> using custom pipe after translating / to \ in Tabby => OK
\\\\.\pipe\pageant.username.verylongcustomhexid
